data/json/monsters/zed_resort.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "mon_zombie_resort_dancer",
```

Comment:
I'm not sure if zombie evolutions carry over with copy-from. You may want to make sure they do, otherwise this facility will get quite easy in the midgame and beyond.

Reply:
I am not sure either, so I will use regular zombie upgrade line.